src/persistentStorage.ts

```diff
@@ -0,0 +1,28 @@
+const cacheName = 'useHTTPcache'
+
+const hasPersistentData = (url: string): boolean => {
+  const urlCache = JSON.parse(localStorage[cacheName] || '{}')
```

Comment:
I think error handling is needed for when parsing fails (probably fallback to empty cache)